src/components/List/index.tsx

```diff
@@ -0,0 +1,171 @@
+import React, { useEffect, useState } from 'react'
+import styled from 'styled-components'
+import { Link } from 'react-router-dom'
+import { makeStyles, Theme, createStyles } from '@material-ui/core/styles'
+import ListMui from '@material-ui/core/List'
+import ListItem from '@material-ui/core/ListItem'
+import ListItemText from '@material-ui/core/ListItemText'
+import Collapse from '@material-ui/core/Collapse'
+import { FixedIcon } from '@gnosis.pm/safe-react-components'
+
+const StyledListItem = styled<any>(ListItem)`
```

Comment:
@mikheevm I need a hand here, for some reason, passing `component` props to StyledListItems throws a TS error. It does not happen if I use ListItem directly.